contracts/SlashIndicator.sol

```diff
@@ -197,6 +203,15 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     emit indicatorCleaned();
   }
 
+  function downtimeSlash(address validator, uint256 count) public override {
+    try IStakeHub(STAKE_HUB_ADDR).downtimeSlash(validator, block.number) {
```

Comment:
For downtime slash, no need to pass `block.number`.

Reply:
Removed